src/mono/browser/build/BrowserWasmApp.targets

```diff
@@ -3,7 +3,8 @@
     <!-- Post Wasm MVP features -->
     <WasmEnableExceptionHandling Condition="'$(WasmEnableExceptionHandling)' == ''">true</WasmEnableExceptionHandling>
     <WasmEnableSIMD Condition="'$(WasmEnableSIMD)' == ''">$(WasmEnableExceptionHandling)</WasmEnableSIMD>
-    <FeaturePerfTracing Condition="'$(FeaturePerfTracing)' == '' and '$(Configuration)' == 'Debug'">true</FeaturePerfTracing>
+    <!-- we are not triggering FeaturePerfTracing in Debug automatically, because it would trigger re-link, making dev-loop slow -->
+    <FeaturePerfTracing Condition="'$(WasmProfilers)' != ''">true</FeaturePerfTracing>
```

Comment:
Is situation when user sets `FeaturePerfTracing=false` and `WasmProfilers=...` valid? Is the override of `FeaturePerfTracing` intentional in that case?